lib/PuppeteerSharp/IConnectionOptions.cs

```diff
@@ -41,5 +41,10 @@ public interface IConnectionOptions
         /// If not <see cref="Transport"/> is set this will be use to determine is the default <see cref="WebSocketTransport"/> will enqueue messages.
         /// </summary>
         bool EnqueueTransportMessages { get; set; }
+
+        /// <summary>
+        /// Affects how responses to <see cref="CDPSession.SendAsync"/> are returned to the caller.
+        /// </summary>
+        bool EnqueueSendAsyncResponses { get; set; }
```

Comment:
I added this as an option, kind of like EnqueueTransportMessages, to let end-users revert to the previous behavior. After fixing the race in CSSCoverage, I'm concerned that users of pptr# will have similar code that assumes CDP responses are handled synchronously, or maybe some scenario that could race but is made worse with this PR.

I can revert this and make this behavior permanent instead, but wanted to try to mitigate broken downstream apps and libs.

Reply:
Same rename here.